app/contexts/MobileCoinDContext.tsx

```diff
@@ -337,41 +337,19 @@ const reducer = (state: MobileCoinDState, action: Action): MobileCoinDState => {
 
 const MobileCoinDContext = createContext<MobileCoinDContextValue>({
   ...initialMobileCoinDState,
-  buildGiftCode: () => {
-    return Promise.resolve();
-  },
-  buildTransaction: () => {
-    return Promise.resolve();
-  },
-  changePassword: () => {
-    return Promise.resolve();
-  },
+  buildGiftCode: () => Promise.resolve(),
+  buildTransaction: () => Promise.resolve(),
+  changePassword: () => Promise.resolve(),
   confirmEntropyKnown: () => {},
-  createAccount: () => {
-    return Promise.resolve();
-  },
+  createAccount: () => Promise.resolve(),
   deleteStoredGiftB58Code: () => {},
```

Comment:
This code is weird: it's not returning an object; rather, it calls an empty block, which by definition implies that an `undefined` value will be returned. For clarity, it would be better writing it as `() => undefined` -- but was that intended? If returning an empty object was needed, the function should have been written as `() => ({})`.